apps/prairielearn/src/components/AssessmentModuleHeading.html.ts

```diff
@@ -0,0 +1,24 @@
+import { html } from '@prairielearn/html';
+
+import type { AssessmentModule } from '../lib/db-types.js';
+
+export function AssessmentModuleHeading({
+  assessment_module,
+}: {
+  assessment_module: AssessmentModule;
+}) {
+  if (
+    !assessment_module.implicit ||
+    (assessment_module.heading === 'Default module' && assessment_module.implicit)
```

Comment:
Nit, this is equivalent:

```suggestion
    !assessment_module.implicit || assessment_module.heading === 'Default '
```

Reply:
This comes from here (which may want to be made a capital `m` to fit the theme of the example course) 
https://github.com/PrairieLearn/PrairieLearn/blob/4b20b51af34b173469b0be56328d6dc0b0a101b5/apps/prairielearn/src/sync/fromDisk/assessmentModules.ts#L52

Reply:
Say less! That was just a bad edit on my part, I dropped the trailing "module" when collapsing lines. I edited my suggestion to correct that; the overall idea of dropping the explicit check for `assessment_module.implicit` in the second branch of the `||` was the important piece of my suggestion.